app/queries/admin/moderators_query.rb

```diff
@@ -4,29 +4,29 @@ class ModeratorsQuery
       state: :trusted
     }.with_indifferent_access.freeze
 
+    VALID_ROLES = %i[banned warned trusted comment_banned].freeze
```

Comment:
This constant doesn't seem to be used anywhere. Should this be `NAMES_FOR_POTENTIAL`, the name you use inside `potential_role_ids`? It's also a somehat less confusing name.

Reply:
Yeah, sorry I committed half of it and didn't commit the rest 🤦